c2/c2-client-bundle/c2-client-service/src/main/java/org/apache/nifi/c2/client/service/C2HeartbeatFactory.java

```diff
@@ -106,6 +106,17 @@ private AgentInfo getAgentInfo(AgentRepositories repos, RuntimeManifest manifest
         return agentInfo;
     }
 
+    private Set<SupportedOperation> getSupportedOperations(RuntimeManifest manifest) {
+        Set<SupportedOperation> supportedOperations;
+        // supported operations has value only in case of minifi, therefore we return empty collection if
```

Comment:
I don't think this command is relevant the code is clear and the comment is unfinished anyway. I would remove it or update it to say AgentManifest instead of minifi. What do you think?